luis-hurtado/tarea-1/styles.css

```diff
@@ -0,0 +1,35 @@
+.magazine-container {
+    width: 800px;
```

Comment:
@luishurtado-globant , please keep alphabetical order on CSS properties.
Also please sort your rules, first IDs, then classes and keep alphabetical order for those